frontend/src/app/core/services/signal-r.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable, EventEmitter } from '@angular/core';
+import * as signalR from '@microsoft/signalr';
+import { environment } from '@env/../environments/environment';
+@Injectable({
+  providedIn: 'root'
+})
+export class SignalRService {
+
+  public hubConnetction: signalR.HubConnection;
+  signalRecieved: EventEmitter<string> = new EventEmitter<string>();
```

Comment:
Use EventEmitter only for communications between components in all other cases - subject.